reader/main.ino

```diff
@@ -59,32 +92,55 @@ void loop() {
   {
     return;
   }
+
   //Show UID on serial monitor
   Serial.println();
   Serial.print(" UID tag :");
   String content= "";
   byte letter;
   for (byte i = 0; i < mfrc522.uid.size; i++) 
   {
-     Serial.print(mfrc522.uid.uidByte[i] < 0x10 ? " 0" : " ");
+     Serial.print(mfrc522.uid.uidByte[i] < 0x10 ? "0" : "");
      Serial.print(mfrc522.uid.uidByte[i], HEX);
-     content.concat(String(mfrc522.uid.uidByte[i] < 0x10 ? " 0" : " "));
+     content.concat(String(mfrc522.uid.uidByte[i] < 0x10 ? "0" : ""));
      content.concat(String(mfrc522.uid.uidByte[i], HEX));
   }
   content.toUpperCase();
   Serial.println();
-  if (content.substring(1) == "8E 39 32 50") //change UID of the card that you want to give access
-  {
-    Serial.println(" Access Granted ");
-    Serial.println(" Welcome Mr.Circuit ");
-    delay(1000);
-    Serial.println(" Have FUN ");
-    Serial.println();
-    statuss = 1;
-  }
+
+  if ((WiFi.status() == WL_CONNECTED)) {
+    WiFiClient client;
+    HTTPClient http;
+  
+    Serial.print("[HTTP] begin...\n");
+    // configure traged server and url
+    http.begin(client, drinkBeerUrl(content.substring(0))); //HTTP
+    http.addHeader("Content-Type", "application/json");
+  
+    Serial.print("[HTTP] POST...\n");
+    // start connection and send HTTP header and body
+    int httpCode = http.POST("{\"id\":\""+content.substring(0)+"\"}");
+  
+    // httpCode will be negative on error
+    if (httpCode > 0) {
+      // HTTP header has been send and Server response header has been handled
+      Serial.printf("[HTTP] POST... code: %d\n", httpCode);
+  
+      const String& payload = http.getString();
+      Serial.println("received payload:\n<<");
+      Serial.println(payload);
+      Serial.println(">>");
+      if (httpCode == HTTP_CODE_OK && payload.length()==14) {
```

Comment:
If I read this correctly, this currently only works because `len("false") != len("true")`
Maybe, we should do a correct payload check at some put in the future :laughing:

Reply:
we could use a errorcode instead.otherwise we need to parse the json. i will see whether to fix it now or later